README.md

```diff
@@ -3,16 +3,70 @@
 Here we are sharing two assets we have developed todate click on either link to dive in further!
 
 
-1. GNURadio - Azure Out of Tree Module
-2. Azure software radio Developer Virtual Machine [Here](./pages/devvm.md)
+1. [GNURadio - Azure Out of Tree Module](./gr-azure-software-radio/docs/README.md)
```

Comment:
Right now ./gr-azure-software-radio/README only has a "Testing" section. I'd suggest we: 

- Move all the info from ./gr-azure-software-radio/docs/README.md into ./gr-azure-software-radio/README.md and delete  ./gr-azure-software-radio/docs/README.md 
- merge the install, build, and test info from the top level README into  ./gr-azure-software-radio/README.md
see if there's anything useful from the existing Testing section in ./gr-azure-software-radio/README.md that we need to keep
- update this doc link to point to ./gr-azure-software-radio/README.md